blazor/treegrid/filtering/excel-like-filter.md

```diff
@@ -0,0 +1,285 @@
+---
+layout: post
+title: Excel Like Filter in Blazor Tree Grid Component | Syncfusion
+description: Checkout and learn here all about Excel like filter in Syncfusion Blazor Tree Grid component and much more details.

```

Comment:
description: Check out here and learn more details about the Excel-like filter in the Syncfusion Blazor Tree Grid component.